spiffe-helper.yaml

```
@@ -0,0 +1,42 @@
+package:
+  name: spiffe-helper
+  version: 0.9.0
+  epoch: 0
+  description: A helper utility for SPIFFE (Secure Production Identity Framework For Everyone) operations.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0" # Required to build a fully static binary
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/spiffe/spiffe-helper.git
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      packages: "./cmd/spiffe-helper"
+      output: spiffe-helper
+
+subpackages:
+  - name: spiffe-helper-compat
+    description: Compatibility package for spiffe-helper
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/
+          ln -s /usr/bin/spiffe-helper "${{targets.subpkgdir}}"/spiffe-helper
```

```suggestion
          ln -sf /usr/bin/spiffe-helper "${{targets.subpkgdir}}"/spiffe-helper
```
to be more consistent with other compats.